lib/src/base-styles/_nx-toast.scss

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../base-styles/nx-alert';
+@use '../scss-shared/nx-container-helpers';
+
+.nx-toast__wrapper {
```

Comment:
The way this element and `nx-toast__container` work to produce the right positioning is clever, but I think it it benefit from an explaining comment

Reply:
> The way this element and `nx-toast__container` work to produce the right positioning is clever, but I think it would benefit from an explaining comment

We added in an explaining comment here: [9e0aa3d](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/9e0aa3dd73d5ec75374886923de3eca358618e80)